### PR TITLE
IncidentAddSystemPassword argument now sensitive.

### PR DESCRIPTION
**Pull request by dem-evgeny**

### CODE REVIEW

File: Scripts/script-IncidentAddSystem.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 commonfields:
   id: IncidentAddSystem
-  version: -1
+  version: 3
```
Comment (dorsha) [inline Scripts/script-IncidentAddSystem.yml]: version should be -1

File: Scripts/script-IncidentAddSystem.yml [hunk 2]
```diff
@@ -36,5 +37,4 @@ args:
 - name: arch
   description: Choose amd64 for 64bit systems and i386 for 32bit systems
 scripttarget: 0
-dependson: {}
-timeout: 0s
+releaseNotes: "Password argument now sensitive."
```
Comment (dorsha) [inline Scripts/script-IncidentAddSystem.yml]: change to "-"

File: Scripts/script-IncidentAddSystem.yml [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+  version: 3
 name: IncidentAddSystem
 script: return executeCommand('addSystem',args);
 type: javascript
 tags:
 - management
 comment: |
   Add a remote system (such as a desktop under investigation) to an investigation (this will allow you to install and agent on the system)
-system: true
+enabled: true
```
Comment (dorsha) [inline Scripts/script-IncidentAddSystem.yml]: is this related?

File: Scripts/script-IncidentAddSystem.yml (hunk 1 above)
Comment (dem-evgeny) [reply Scripts/script-IncidentAddSystem.yml]: fixed

File: Scripts/script-IncidentAddSystem.yml (hunk 2 above)
Comment (dem-evgeny) [reply Scripts/script-IncidentAddSystem.yml]: changed

File: Scripts/script-IncidentAddSystem.yml (hunk 3 above)
Comment (dem-evgeny) [reply Scripts/script-IncidentAddSystem.yml]: changed
